### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by DerDennisOP**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -197,6 +197,11 @@
                   nixpkgs.overlays = [
                     (_self: super: {
                       libgit2 = super.libgit2.overrideAttrs { doCheck = false; };
+                      nixos-rebuild = super.nixos-rebuild.overrideAttrs (old: {
+                        patches = (old.patches or []) ++ [
+                          ./modules/nom.patch
```
Comment (ahuston-0) [inline flake.nix:202]: so is this passing everything through nix-output-monitor? Is this a permanent or temp patch?

**Review by ahuston-0 — COMMENTED** [review]
pls see comment.

File: flake.nix (hunk 1 above)
Comment (DerDennisOP) [reply flake.nix:202]: yes, permanent but I will test it now

Comment (ahuston-0) [reply flake.nix:203]: okay yeah pls do and lmk

Comment (DerDennisOP) [reply flake.nix:203]: dosnt work, I will look into it later

**Review by ahuston-0 — APPROVED** [review]
looks good. global impacts limited to aarch64 it looks like so deferring second approval to @FraFrieFa

**Comment by DerDennisOP** [comment]
frafrifra knows about it

**Pull request merged**